spec/controllers/api/bot_auth_spec.rb

```diff
@@ -8,8 +8,8 @@
     let(:device) { FactoryGirl.create(:device) }
 
     it 'authorizes using MeshBlu UUID / Token' do
-      @request.headers["bot_token"] = device.token
-      @request.headers["bot_uuid"]  = device.uuid
+      @request.headers["HTTP_BOT_TOKEN"] = device.token
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.